packages/iobio-react-components/README.md

```diff
@@ -2,14 +2,52 @@
 
 ## Setup + Local Development
 
+### Requirements:
+
+- PNPM 9+
+- Node 20+
+
 - Install dependencies: `pnpm i`
```

Comment:
should this be `ci`? or does pnpm figure out all that npm `ci` vs `i` stuff itself

Reply:
PNPM is already managing that.

https://pnpm.io/cli/install
https://stackoverflow.com/questions/70154568/pnpm-equivalent-command-for-npm-ci